authors/apps/articles/views.py

```diff
@@ -23,10 +26,15 @@ class ListCreateArticles(generics.ListCreateAPIView):
     """
     queryset = Article.objects.all()
     serializer_class = ArticleSerializer
-    permission_classes = (IsAuthenticated, )
+    permission_classes = (IsAuthenticatedOrReadOnly, )
```

Comment:
@3Nakajugo This doesn't  seem to work. Look into it. I blocked from accessing the endpoint without the token.